_posts/2022-11-22-mlops-competition-recap.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+author: Hamza Tahir
+title: "ZenML's Month of MLOps Recap"
+description: "The ZenML MLOps Competition ran from October 10 to November 11, 2022, and was a wonderful expression of open-source MLOps problem-solving."
+category: zenml
+tags: zenml competition mlops
+publish_date: November 22, 2022
+date: 2022-11-22T00:02:00Z
+thumbnail: /assets/posts/competition-recap/Final.png
+image:
+  path: /assets/posts/competition-recap/Final.png
```

Comment:
I made a thumbnail GIF for this. Let's put this in?

![Month of mlops 2022](https://user-images.githubusercontent.com/6821286/203242204-ca9a7392-2aba-4533-9e14-9479787b916d.gif)